lib/postprocessing/image.js

```diff
@@ -634,13 +634,21 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
 
 function convertSvg(img, instructions) {
     // ImageMagick automatically keeps aspect ratio
-    // Only using width because ImageMagick will use the smallest value whether it be width or height
-    const width = instructions.width || SVG_DEFAULT_WIDTH;
+    let imageSize = SVG_DEFAULT_WIDTH;
+
+    if(instructions.width && instructions.height) {
+        // image magick will keep aspect ratio
+        imageSize = `${instructions.width}x${instructions.height}`;
+    } else if(instructions.width) {
+        imageSize = `${instructions.width}x`;
```

Comment:
The `x` is left for clarity in the instruction, to know we got width only.

Reply:
do we want to log wether we only got with or height or got both instead of having to examine the cmd?